src/views/Catalog.vue

```diff
@@ -160,7 +160,7 @@ export default {
     catalogAsFc () {
       return {
         type: 'FeatureCollection',
-        features: this.items
+        features: this.items.map(i => i._rawJson)
```

Comment:
Is this actually required? The STAC class is a decorator to the STAC object, so I'd assume you can simply pass it over.

Another side note: To keep the code base consistent, I'd like to see `;` at the end of the lines. :-) Actually, I thought eslint would ask for them, but maybe I need to add that.

Reply:
I added this when having problems with clearing the stachlayer, I thought perhaps the class object might be having some side effects, so happy to remove it.
And yep add an eslint rule and I'll follow it 🙂

Reply:
Ah, okay, then we should investigate the side effects and try to avoid those :-) Any details?

Reply:
I've added eslint rules in the v3 branch, it should fix most issues automatically once you run `npm run lint`.

Reply:
Have addressed comment. And the weirdness with the stac-layer dependency seems to have disappeared, maybe because versions got pinned somewhere.